README.md

```diff
@@ -259,37 +366,40 @@ Let's fire up the OCaml top-level:
 
 ```ocaml
 $ ocaml
-        OCaml version 4.02.2+multicore-dev0
+OCaml version 5.0.0~beta1
 
-# effect E : unit;; 
-effect E : unit  (* E is a nullary effect that returns unit *)
+# open Effect;;
+# type _ Effect.t += E : unit Effect.t;;
+type _ Stdlib.Effect.t += E : unit Effect.t
 # let f () = perform E;;
 val f : unit -> unit = <fun>
 # f ();;
-Exception: Unhandled.
-# match f () with (* alternative syntax for effect handlers *)
-  | () -> "done"                              (* value case *)
-  | effect E k -> continue k ();;            (* effect case *)
-- : string = "done"
+Exception: Stdlib.Effect.Unhandled(E)
+# open Effect.Deep;;
+# try_with f () {effc = (fun (type c) (eff: c Effect.t) ->
+      match eff with
+      | E -> Some (fun (k: (c,_) continuation) -> continue k ())
+      | _ -> None
+  )};;
+- : unit = ()
 ```
 
-An effect system for OCaml 
-[is in development](https://www.youtube.com/watch?t=2035&v=z8SI7WBtlcA&feature=youtu.be). 
```

Comment:
This is still true.

Reply:
If this is aimed at being a tutorial for the initial release of effects, does it still make sense to say it is "in development"?

It seems out of place and doesn't flow well